### PR TITLE
add id`s

### PR DESCRIPTION
**Pull request by vadim9999**

### CODE REVIEW

File: src/views/Examples/Examples.js [hunk 1]
```diff
@@ -28,11 +28,11 @@ class Examples extends Component {
             <h3>Checkbox</h3>
               <Checkbox>Check</Checkbox>
             <h3>DeleteLink</h3>
-              <DeleteLink url="/examples">Link</DeleteLink>
+              <DeleteLink id = "123" url="/examples">Link</DeleteLink>
```
Comment (atherdon) [inline src/views/Examples/Examples.js]: for delete and edit link it should be similar to this
url="/examples/delete/:id"

Comment (atherdon) [reply src/views/Examples/Examples.js]: i saw your latest changes. did we have an `id` prop at our components EditLink, DeleteLink? i think we should have it, in order to generate right url.

**Comment by vadim9999** [comment]
OldRouter doesn't work

**Comment by atherdon** [comment]
> OldRouter doesn't work

Ok, let's try to fix it together - are you ok with this plan?

**Comment by atherdon** [comment]
merged

**Pull request merged**